lib/ui/surveydetail/survey_question_content.dart

```diff
@@ -1,11 +1,14 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_rating_bar/flutter_rating_bar.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_survey/ui/surveydetail/single_selectable_rating_bar.dart';
+import 'package:flutter_survey/utils/extension/iterable_ext.dart';
 
 import '../../model/question.dart';
 
 const starEmoji = "⭐️️";
 const heartEmoji = "❤️️";
+const smileys = {0: "😡", 1: "😕", 2: "😐", 3: "🙂", 4: "😄"};
```

Comment:
What do you think about marking as private for these constants? Or we should move it to a constant file if we want to public them 🙏

Reply:
Done. https://github.com/nimblehq/ic-flutter-avishek/pull/61/commits/a6d813955431c1c0e0cb5abfc7ef39a7903a462b